### PR TITLE
[SHEPHERD] Add utils file for headers

### PR DESCRIPTION
**Pull request by brandonxxlee**
Each header is autoenumerated

### CODE REVIEW

File: shepherd/Utils.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+from enum import Enum, unique
+
+class Header(Enum):
+    # Used to autoenumerate
```
Comment (FIREdog5) [inline shepherd/Utils.py]: Seems like this should be described with a doc string, but it really doesn't matter.

File: shepherd/Utils.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    def __new__(cls):
+        value = len(cls.__members__) + 1 # pylint: disable=no-member
+        obj = object.__new__(cls)
+        obj._value_ = value # pylint: disable=protected-access
+        return obj
+
+@unique # pylint: disable=invalid-name
+class SHEPHERD_HEADER(Header):
```
Comment (FIREdog5) [inline shepherd/Utils.py]: It seems like there should be some sort of documentation associated with these headers... again tho, if it's just for us then it doesn't really matter.

**Review by FIREdog5 — DISMISSED** [review]
looks good enough to me.

**Review by bodiddles — DISMISSED** [review]
lgtm

**Review by JHGZhou — DISMISSED** [review]
Might have to consider not using auto, but is usable for now

**Review by JHGZhou — APPROVED** [review]
I'm lowkey approving some of my own changes. But thats fine

**Pull request merged**